creator-node/src/routes/nodeSync.js

```diff
@@ -224,12 +236,20 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint, dbOnlySync
   }
 
   try {
-    // Fetch data export from creatorNodeEndpoint for given walletPublicKeys
+    // Query own latest clockValue and call export with that value + 1; export from 0 for first time sync
+    const cnodeUser = await models.CNodeUser.findOne({
+      where: { walletPublicKey: walletPublicKeys[0] }
+    })
+    const localMaxClockVal = (cnodeUser) ? cnodeUser.clock : -1
```

Comment:
so to clarify, this is the max clock value on the secondary, which is sent over to the primary node - the primary actually performs the range calculation and sends the data back

Reply:
might be worth renaming the variable here for extra clarity but meh

Reply:
yep thats exactly what happens, can add comments to make more clear

Reply:
lets do that before merging

Reply:
good idea, added